es-bs-base/scss/vue/components/_modal.scss

```diff
@@ -0,0 +1,7 @@
+@use "../../variables" as v;
```

Comment:
Can this be something besides `v`? Maybe `vueVariables` or something

Reply:
Without the `v`, it would resolve to "variables".

Since both `bootstrap-vue` variables and the bootstrap variables are referenced, I opted for the convention of using the alias `v` for root-level variables, and `variables` for "local-ish" variables.

The `v` here is pointing to the root SASS variables, not the vue ones.